ibm/service/kms/resource_ibm_kms_key_rings.go

```diff
@@ -35,6 +36,13 @@ func ResourceIBMKmskeyRings() *schema.Resource {
 				Description:  "User defined unique ID for the key ring",
 				ValidateFunc: validate.InvokeValidator("ibm_kms_key_rings", "key_ring_id"),
 			},
+			"force_delete": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "set to true to force delete this key ring. This allows key ring deletion as long as all keys inside have key state equals to 5(destroyed)",
```

Comment:
Suggestion:

set to true to force delete this key ring. This allows non empty key ring deletion as long as all keys inside the key ring are in 5 (destroyed). Keys are moved to the default key ring.

Reply:
Agree, description should indicate keys will be moved

Reply:
done